DESCRIPTION.md

```diff
@@ -8,7 +8,7 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v3.12.1(TBD)
+- v3.12.1(August 20,2024)
   - Fixed a bug that session token is logged when renewing session.
   - Fixed a bug that disabling client telemetry does not work.
   - Fixed a bug where `login_timeout` when passed as string raised `TypeError` during login retry step.
```

Comment:
Fixed a bug where passing ``login_timeout`` as a string raised a ``TypeError`` during the login retry step.